scripts/mbedtls_framework/psa_compliance.py

```diff
@@ -1,16 +1,19 @@
 """Run the PSA Crypto API compliance test suite.
-Clone the repo and check out the commit specified by PSA_ARCH_TEST_REPO and PSA_ARCH_TEST_REF,
-then compile and run the test suite. The clone is stored at <repository root>/psa-arch-tests.
-Known defects in either the test suite or mbedtls / TF-PSA-Crypto - identified by their test
-number - are ignored, while unexpected failures AND successes are reported as errors, to help
+
+Clone the psa-arch-tests repo and check out the specified commit.
+The clone is stored at <repository root>/psa-arch-tests.
+Check out the commit specified by the calling script and apply patches if needed.
```

Comment:
I think this is now a command-line argument.

Reply:
Normally the commit comes from the calling script. It can be overridden by a command line argument, but I don't think the general description needs to list all the settings that can be overridden on the command line.